src/tools/ApiChecker/IdentifyMissingTests.cs

```diff
@@ -0,0 +1,168 @@
+/*
+ * MIT License
+ * 
+ * Copyright (c) Microsoft Corporation.
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using CommandLine;
+
+namespace ApiChecker
+{
+#pragma warning disable SA1201 // Elements should appear in the correct order
```

Comment:
Why don't we honor this rule?

Reply:
It's kind of _stupid_. But that `Options` class is `internal` so if we followed that rule, it'd have to be _beneath_ the `Run` method, which doesn't really make sense.

Reply:
I would follow that rule...

Reply:
Well, rules are rules :-).

I moved it to the bottom, same as with `ScaffoldTest`. Consistency.